client/src/app/site/mediafiles/components/mediafile-list/mediafile-list.component.ts

```diff
@@ -459,6 +461,16 @@ export class MediafileListComponent extends BaseListViewComponent<ViewMediafile>
         });
     }
 
+    public downloadMultiple(mediafiles: ViewMediafile[] = this.dataSource.source): void {
+        /**
+         * TODO: naming the files is discussable
+         */
+        const eventName = this.configService.instant<string>('general_event_name');
+        const dirName = this.directory?.filename ?? this.translate.instant('Files');
+        const archiveName = `${eventName} - ${dirName}`.trim();
+        this.repo.downloadArchive(archiveName, mediafiles);
```

Comment:
Does this handle folders recursively. I suppose coosing a folder, that the content of it is downloaded, too.

Reply:
If it is not in scope, it's ok for me.

Reply:
https://github.com/OpenSlides/OpenSlides/pull/5505#issuecomment-672828945

Reply:
Ok, please fix the other remarks. The rest looks good!